featurebyte/api/api_object_util.py

```diff
@@ -146,20 +145,6 @@ def __getattribute__(self, item: str) -> Any:
         return super().__getattribute__(item)
 
 
-def is_server_mode() -> bool:
-    """
-    Check if the code is running in server mode. Server mode is used when running the SDK code inside
-    featurebyte worker.
-
-    Returns
-    -------
-    bool
-        True if the code is running in server mode
-    """
-    sdk_execution_mode = os.environ.get("FEATUREBYTE_SDK_EXECUTION_MODE")
-    return sdk_execution_mode == "SERVER"
```

Comment:
Move this to the upper level as we need to import this in `featurebyte/config.py`.